webapp/src/reducers/index.js

```diff
@@ -25,6 +26,20 @@ const createModalVisible = (state = false, action) => {
     }
 };
 
+const createModalEmpty = (state = '', action) => {
```

Comment:
This can probably be combined with the reducer below so we only have one thing to check for the create issue modal.

Reply:
Good point, done. Much nicer now. I also made it more idiomatic reduxy.